showcase-v1alpha2/src/index.ts

```diff
@@ -0,0 +1,64 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @namespace google
+ */
+/**
+ * @namespace google.cloud
+ */
+/**
+ * @namespace google.cloud.showcase
+ */
+/**
+ * @namespace google.cloud.showcase.v1alpha2
+ */
+
+'use strict';
+
+// Import the clients for each version supported by this package.
+import {EchoClient} from './v1alpha2';
+
+export {EchoClient as EchoClientV1alpha2} from './v1alpha2/';
```

Comment:
Is this for compat with what we have now? Seems pretty ugly (we used to have `.v1alpha2.EchoClient`).

That said, if this is there for compatibility with the _status quo ante_, obviously leave it as-is.

Reply:
I need to think if I can make it look like `v1alpha2.EchoClient`.